src/org/ods/orchestration/phases/FinalizeOdsComponent.groovy

```diff
@@ -137,41 +137,39 @@ class FinalizeOdsComponent {
             throw new RuntimeException ("Deploymentmeans for repo ${repo.id}" +
```

Comment:
see internal thread - maybe we should remove this check as well?! .. can you try to create a component W/O deployment . and transport it thru?!